source/blog/2023-10-11-orm3-beta1-dbal4-rc1-future-plans.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "ORM 3.0 Beta 1, DBAL 4 RC 1 and future plans"
+authorName: Benjamin Eberlei
+authorEmail: kontakt@beberlei.de
+permalink: /2023/10/11/orm3-beta1-dbal4-rc1-future-plans.html
+---
+
+We have released the first beta of the long awaited Doctrine ORM 3 and a
+release candidate of DBAL 4. 
+
+The target audience for these releases are framework integration and extension
+library authors. ORM 3 is not yet production ready and the APIs may change.
+
+Our goal is to release ORM 3.0 as soon as possible and to gather feedback from
+greenfield project authors first. 
+
+This beta release is the result of a lot of work by many contributors,
+especially Grégoire, Alexander, Claudio and Matthias on ORM and Sergei on DBAL.
+To iron out the final details, we met in Düsseldorf for a Doctrine Core Team
+meeting, generously funded by our sponsors through OpenCollective and GitHub.
+We also welcomed Matthias as the latest member of the Doctrine Core Team.
+
+### Continued ORM 2 support and forward compatibility
+
+We will maintain the latest branch of the 2 line in ORM for at least another 2
+years, possibly longer, to give you enough time to upgrade and us more time to
+learn from upgrader feedback and improve forward compatibility.
+
+This means that we will be making ORM 2.x work with newer versions of PHP,
+fixing security bugs, and introducing layers and features that help with
+forward compatibility in the upgrade path to ORM 3. 
+
+Current users of ORM 2 are strongly encouraged to stay on version 2, as we are
+still working on replacement APIs and forward compatibility, and do not intend
+to ship them all with ORM 3.0, but with later versions.  
```

Comment:
I would rephrase that as this sounds like we're actively holding back people that want to try the new major release. And if nobody dares to upgrade, we probably won't get valuable feedback on the upgrade path and the removed features.

The release is flagged as beta, which should tell people that they shouldn't use it in production yet. And since we're saying that we're going to maintain ORM 2 for some time, I'd rather emphasize that there's no urgency to upgrade just yet.